### PR TITLE
Update socli.py

### PR DESCRIPTION
**Pull request by manish70668**
history is added to retrieve the last 15 commands executed by the user.  Issue #234

### CODE REVIEW

File: socli/socli.py [hunk 1]
```diff
@@ -319,7 +319,16 @@ def main():
     """
     The main logic for how options in a command is checked.
     """
+    history=[]
```
Comment (manish70668) [inline socli/socli.py:322]: Storing History

**Review by gautamkrishnar — CHANGES_REQUESTED** [review]
Hi @manish70668 Can you please explain what are you trying to achieve here. You are taking the command line arguments and printing it to the console. That's not what it's meant in issue #234. Please read the issue first before contributing. The issue is about showing the previous queries as autocompletion on the shell. Not about printing argv to the console.

**Comment by gautamkrishnar** [comment]
@manish70668 closing this one, if you are interested please submit a new PR with the correct implementation.

**Comment by manish70668** [comment]
Hi @gautamkrishnar , I have misunderstood the requirement, I will work on it and will submit the new PR for the same. Thanks!!!

**Pull request closed without merging**